src/ToDoList.js

```diff
@@ -11,90 +11,163 @@ let taskData = [
     id: 1,
     taskName: "sample task",
     taskDescription: "sample description",
+    isComplete: false,
+  },
+  {
+    id: 2,
+    taskName: "sample task",
+    taskDescription: "sample description",
+    isComplete: true, 
   },
 ];
-
 const ToDoList = () => {
   const [isOpen, setOpen] = useState(false);
-  const [taskCard, setTaskCard] = useState(taskData);
+  const [taskList, setTaskCard] = useState(taskData);
+ 
+  const updateStatusHandler = (id) => {
+    console.log(id);
+    var newTaskData = [];
+    for (var i = 0; i < taskList.length; i++) {
+      if (taskList[i].id == id) {
```

Comment:
```suggestion
      if (taskList[i].id === id) {
```
Use `===` for more information [please find the details here ](https://codeahoy.com/javascript/2019/10/12/==-vs-===-in-javascript/#:~:text=The%20difference%20between%20%3D%3D%20and%20%3D%3D%3D%20is%20that%3A,the%20two%20variables%20being%20compared.)